crates/turborepo-lib/src/config/repo.rs

```diff
@@ -21,50 +21,57 @@ pub struct RepoConfig {
 
 #[derive(Debug, Deserialize, Serialize, Clone, PartialEq, Eq, Default)]
 struct RepoConfigValue {
-    apiurl: Option<String>,
-    loginurl: Option<String>,
-    teamslug: Option<String>,
-    teamid: Option<String>,
+    #[serde(rename = "apiurl")]
```

Comment:
what are all the renames for?

Reply:
`loginurl` is not idiomatic Rust naming, so I figured it's better to have `login_url` in the Rust code then rename it to `loginurl` via Serde

Reply:
that makes sense, but I wasn't sure where the non-idiomatic versions (e.g. `loginurl`) are used and why we can't replace that?

Reply:
It's used in the config, and we want to avoid accidentally signing people out/unlinking them when we move from the Go to the Rust version.